content/education/manage-coursework-with-github-classroom/teach-with-github-classroom/create-a-group-assignment.md

```diff
@@ -116,7 +116,7 @@ Optionally, you can automatically grade assignments and create a space for discu
 
 ### Creating a pull request for feedback
 
-{% data reusables.classroom.you-can-create-a-pull-request-for-feedback %}
+{% data reusables.classroom.you-can-create-a-pull-request-for-feedback %} For more information on leaving feedbacks in a pull request, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/leave-feedback-with-pull-requests)".
```

Comment:
```suggestion
{% data reusables.classroom.you-can-create-a-pull-request-for-feedback %} For more information on leaving feedback in a pull request, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/leave-feedback-with-pull-requests)."
```

Thanks so much for adding this here! I've just suggested a couple small typo fixes 🙂 according to the styling convention we follow ([Chicago Manual of Style](https://www.chicagomanualofstyle.org/home.html)), the period should technically go within the quotes at the end of a sentence instead of outside them (but honestly, I think it looks a little strange 🙃)